consensus/hotstuff/voteaggregator/vote_aggregator_v2.go

```diff
@@ -185,8 +185,25 @@ func (va *VoteAggregatorV2) AddBlock(block *model.Proposal) error {
 	return nil
 }
 
-func (va *VoteAggregatorV2) InvalidBlock(block *model.Proposal) error {
-	panic("implement me")
+// InvalidBlock notifies the VoteAggregator about an invalid proposal, so that it can process votes for the invalid
+// block and slash the voters.
```

Comment:
lets document the expected error returns:

```suggestion
// block and slash the voters.
// Expected error returns during normal operations:
//  * mempool.DecreasingPruningHeightError if proposal's view has already been pruned 
```

could you also add the same comment to the interface, please https://github.com/onflow/flow-go/blob/e371dc223162a1f7496980345ecfae507d88023a/consensus/hotstuff/vote_aggregator_v2.go#L50